.travis.yml

```diff
@@ -22,10 +22,13 @@ before_install:
     - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
     - phpunit --self-update
     - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+    - npm install -g swagger-cli
```

Comment:
They don't need it to develop. It's just an extra check we do in the CI to be sure that the output of the Swagger generator is correct.
If we used a go validator (there is a goswagger is able to generate a Swagger file), we would not have versioned a go setup in the repo just to use this program.